java-test-samples/java-appsync-sam/src/test/java/com/example/MockTest.java

```diff
@@ -0,0 +1,404 @@
+package com.example;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import software.amazon.awssdk.services.appsync.AppSyncClient;
+import software.amazon.awssdk.services.appsync.model.EvaluateMappingTemplateRequest;
+import software.amazon.awssdk.services.appsync.model.EvaluateMappingTemplateResponse;
+
+public class MockTest {
+
+    private static String locationContext = "{\"arguments\":\n    {\n        \"locationid\": \"1234567890\",\n        \"name\": \"Location Name\",\n        \"description\": \"Location Description\",\n        \"imageUrl\": \"https://www.example.com/image.jpg\"\n    },\n\"result\": {\n    \"locationid\": \"1234567890\",\n    \"imageUrl\": \"https://www.example.com/image.jpg\",\n    \"name\": \"Location Name\",\n    \"description\": \"Location Description\",\n    \"timestamp\": \"2023-01-01T00:00:00.000Z\"\n}}";
```

Comment:
Move long JSON strings to a file and read the file using ObjectMapper. Or prefer using multi-line strings with Java 17.

Reply:
done